src/main/java/org/graylog/plugins/usagestatistics/UsageStatsPeriodical.java

```diff
@@ -74,8 +71,23 @@ protected UsageStatsPeriodical(UsageStatsConfiguration config,
 
     protected abstract URL getUrl();
 
+    protected boolean isEnabled() {
+        if (!config.isEnabled()) {
+            return false;
+        }
+
+        final UsageStatsOptOutState optOutState = clusterConfigService.get(UsageStatsOptOutState.class);
+
+        return optOutState == null || !optOutState.isOptOut();
```

Comment:
When using `ClusterConfigService.getOrDefault()` in the previous line, this return statement could be simplified to `return !optOutState.isOptOut();`